jenkins/ci.suse.de/templates/cloud-gating-template.yaml

```diff
@@ -15,9 +15,9 @@
       - validating-string:
           name: cloud_env
           default: '{cloud_env|}'
-          regex: '[A-Za-z0-9-_]+'
```

Comment:
This works in PCRE, but others only allow a literal dash at the end or beginning. Lets hope it works in Jenkins...